htdocs/search.php

```diff
@@ -1490,7 +1490,13 @@
 
     $count = $caches_per_page;
     if (isset($_REQUEST['count'])) { // Ocprop
-        $count = floor($_REQUEST['count'] + 0);
+        if ($_REQUEST['count'] == 'max') {
```

Comment:
It's saver to use `===` almost always. Example: `true == 'max'` is `true` not `false`. And `===` is also faster than `==` because no type conversion needs to be done.